app/controllers/concerns/cohort_authorization.rb

```diff
@@ -21,11 +21,11 @@ def require_can_access_cohort!
     end
 
     def require_can_edit_cohort!
-      current_user.can_manage_cohorts? || current_user.can_edit_cohort_clients?
+      current_user.can_edit_some_cohorts
```

Comment:
Usually these `require..!` methods return not_authorized or true (as on line 27) does it make sense to update this one?  If not, then I think we can replace all calls to `require_can_edit_cohort!` with `current_user.can_edit_some_cohorts`